src/test/java/bio/terra/service/resourcemanagement/google/GoogleProjectServiceOnDemandTest.java

```diff
@@ -1,29 +1,65 @@
 package bio.terra.service.resourcemanagement.google;
 
+import bio.terra.app.configuration.ConnectedTestConfiguration;
 import bio.terra.app.model.GoogleRegion;
+import bio.terra.buffer.model.ResourceInfo;
+import bio.terra.common.EmbeddedDatabaseTest;
 import bio.terra.common.category.OnDemand;
+import bio.terra.common.fixtures.ConnectedOperations;
+import bio.terra.model.BillingProfileModel;
+import bio.terra.service.auth.iam.IamProviderInterface;
+import bio.terra.service.resourcemanagement.BufferService;
+import com.google.api.services.cloudresourcemanager.model.Project;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.context.junit4.SpringRunner;
 
 @RunWith(SpringRunner.class)
-@AutoConfigureMockMvc
 @SpringBootTest
+@AutoConfigureMockMvc
+@ActiveProfiles({"google", "connectedtest"})
 @Category(OnDemand.class)
+@EmbeddedDatabaseTest
 public class GoogleProjectServiceOnDemandTest {
 
-  @Autowired private GoogleProjectService projectService;
+  @Autowired private BufferService bufferService;
```

Comment:
I'm curious about why you had to change this ondemand test?  Was there an API change?

Reply:
It was a pre-RBS test that didn't work without getting updated. I used it to debug the stairway/appengine issue, but decided to leave it as an ondemand test since I don't think it provides a lot of value on its own.